scripts/get.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+## Derived from https://github.com/goreleaser/get/blob/03c4bfde763b30bfe270892ab3ff74949d8e9351/get
+
+set -e
+
+die () {
+    echo >&2 "$@"
+    exit 1
+}
+
+TAR_FILE="/tmp/ike.tar.gz"
+RELEASES_URL="https://github.com/maistra/istio-workspace/releases"
+
+last_version() {
+  curl -sL -o /dev/null -w %{url_effective} "$RELEASES_URL/latest" |
+    rev |
+    cut -f1 -d'/'|
+    rev
+}
+
+download() {
+  rm -f "$TAR_FILE"
+  curl -s -L -o "$TAR_FILE" \
+    "$RELEASES_URL/download/$VERSION/ike_${VERSION:1}_$(uname -s)_$(uname -m).tar.gz"
+}
+
+test -z "$VERSION" && VERSION="$(last_version)"
+test -z "$VERSION" && {
+  die "Unable to get ike version. You can still try to download manually from $RELEASES_URL"
+}
+test -z "$TMPDIR" && TMPDIR="$(mktemp -d --suffix=-ike-${VERSION})"
+
+download
+tar -xf "$TAR_FILE" -C "$TMPDIR"
+
+echo "Downloaded ike binary to $TMPDIR"
+echo -e "You can add it to your path by typing following in your terminal:\n$ export PATH=\"\$PATH:$TMPDIR\""
```

Comment:
TMPDIR is deleted on reboot etc isn't it?

Reply:
Right. Should add a flag or variable so that the user can define the location himself?

Reply:
yea.. the alternative is to require root and place it in /usr/local/bin or something...

Reply:
e424305 brings the following behaviour:

```
╰─ cat scripts/get.sh | bash -s -- --dir=./dist
Downloaded ike binary (v0.0.1) to ./dist
Make sure it's on your $PATH.
```

Good enough?